resources/forwarding_rule.rb

```diff
@@ -205,6 +205,9 @@ def update
             # TODO(nelsonjr): Check w/ Chef... can we print this in red?
             puts # making a newline until we find a better way TODO: find!
             compute_changes.each { |log| puts "    - #{log.strip}\n" }
+              if (@current_resource.target != @new_resource.target)
+                target_update(@current_resource)
+              end
             update_req =
```

Comment:
This is the same deal as the puppet one, where we should only do the partial and not the full